platform/android/app/src/main/kotlin/coop/polypoly/polypod/homescreen/HomeScreenViewModel.kt

```diff
@@ -105,25 +109,49 @@ data class Footer(
 
 class HomeScreenViewModel {
     fun getSectionModels(
+        context: Context,
         onFeatureSelected: (String) -> Unit
     ): List<SectionModel> {
         return FeatureStorage.categories.map { category ->
             SectionModel(
                 category.name,
-                SectionType.fromCategoryType(category.category),
+                SectionType.fromCategoryType(category.id),
```

Comment:
Right, and again here it's fromCategory*Type* and the argument is category *id*. So maybe we want to say *id* instead of type here?